ext/drsyms/drsyms_elf.c

```diff
@@ -352,7 +354,9 @@ drsym_obj_symbol_offs(void *mod_in, uint idx, size_t *offs_start OUT,
     elf_info_t *mod = (elf_info_t *) mod_in;
     if (offs_start == NULL || mod == NULL || idx >= mod->num_syms || mod->syms == NULL)
         return DRSYM_ERROR_INVALID_PARAMETER;
-    if (mod->syms[idx].st_value == 0) {
+    if ((mod->syms[idx].st_value == 0 &&
```

Comment:
suggest: a comment asking to keep consistent with symbol_is_import() in the core